detection_rules/schemas/__init__.py

```diff
@@ -212,6 +212,12 @@ def migrate_to_8_4(version: Version, api_contents: dict) -> dict:
     return strip_additional_properties(version, api_contents)
 
 
+@migrate("8.5")
```

Comment:
May need to verify the invalidation of _all_ restricted fields and rule types for downgrades. Maybe punt that to a separate issue

Reply:
Noted and agreed. Added an issue for this that can be solved in a separate PR.

* https://github.com/elastic/detection-rules/issues/2320